test/test_mkldnn.py

```diff
@@ -703,6 +703,14 @@ def test_0_dimension_tensor(self):
         with self.assertRaisesRegex(RuntimeError, "must match"):
             torch.rand([5]).to_mkldnn() + torch.rand([0]).to_mkldnn()
 
+        C = 7
+        m = mkldnn_utils.to_mkldnn(torch.nn.Conv2d(C, C, 3))
+        x = torch.randn(0, C, C, 8, dtype=torch.float)
+        out_eager = m(x)
```

Comment:
hasn't `m` already been converted to mkldnn?
are we testing here `eager` vs `mkldnn` ?

Reply:
yea, good catch